lxdock/guests/base.py

```diff
@@ -175,6 +182,10 @@ def copy_directory(self, host_path, guest_path):
         self.run(['tar', '-xf', guest_tar_path, '-C', str(guest_path)])
         self.run(['rm', '-f', str(guest_tar_path)])
 
+    def is_provisioned(self):
+        """ Returns a boolean indicating if the container is provisioned. """
+        return self.lxd_container.config.get('user.lxdock.provisioned') == 'true'
+
```

Comment:
@hsoft Forget my question. I understood it wrong. I was thinking the Provisioner class completely jumped over the provisioning for provisioned containers, but in fact it only jumps over the setup. This is definitely good to do.